### ISSUE TITLE
Console crashes when creating operand with a multi-version CRD

### ISSUE THREAD

**Issue by luksa** [openshift/console#6797]
We've recently introduced an additional version into our CRD and updated the CSV to use the new version. This broke the creation of the CR via the web console. The following error occurs:
```
vendors~main-chunk-415d2ffa1b0b13eaafc2.min.js:161175 Error: Could not find a definition for #/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta.
    at N (vendors~create-binding~create-operand~create-service-instance-chunk-1363f120289e33f90c46.min.js:1)
    at e (vendors~create-binding~create-operand~create-service-instance-chunk-1363f120289e33f90c46.min.js:1)
    at vendors~create-binding~create-operand~create-service-instance-chunk-1363f120289e33f90c46.min.js:1
    at Array.reduce (<anonymous>)
    at e (vendors~create-binding~create-operand~create-service-instance-chunk-1363f120289e33f90c46.min.js:1)
    at t.getDefaultFormState (vendors~create-binding~create-operand~create-service-instance-chunk-1363f120289e33f90c46.min.js:1)
    at t.value (vendors~create-binding~create-operand~create-service-instance-chunk-1363f120289e33f90c46.min.js:1)
    at new t (vendors~create-binding~create-operand~create-service-instance-chunk-1363f120289e33f90c46.min.js:1)
    at Wi (vendors~main-chunk-415d2ffa1b0b13eaafc2.min.js:161175)
    at ma (vendors~main-chunk-415d2ffa1b0b13eaafc2.min.js:161175)
```

I've been investigating the issue and it seems to happen with any CRD that contains more than one version. As far as I could gather from debugging the javascript code, it seems that an empty `definitions` value is passed into the `findSchemaDefinition` function. Could this be caused by the fact that in multi-version CRDs, the schema is in `CRD.spec.versions.schema` and not in `CRD.spec.validation` (Note: we're using apiextension.k8s.io/v1beta1 for the CRD)? 

What's interesting is that if I add `x-kubernetes-preserve-unknown-fields` to the `spec` field in the CRD's schema, the issue goes away. Not sure why changing the schema for the `spec` would effect the `metadata` schema. 

Is this a known issue? If not, I can create a minimal reproducer.

Environment:
```
$ crc version
CodeReady Containers version: 1.16.0+bf72d3a
OpenShift version: 4.5.9 (embedded in binary)
```

**Comment by spadgett** [comment]
This appears to be the same bug that is fixed by #6821. @TheRealJon can you confirm?

**Comment by spadgett** [comment]
I've confirmed with @TheRealJon this is indeed fixed by #6821. See also https://bugzilla.redhat.com/show_bug.cgi?id=1884613

/close
